include/seastar/http/client.hh

```diff
@@ -180,15 +185,20 @@ class client {
     using connection_ptr = seastar::shared_ptr<connection>;
 
     future<connection_ptr> get_connection();
+    future<connection_ptr> make_connection();
     future<> put_connection(connection_ptr con);
     future<> shrink_connections();
 
+    struct force_new_tag {};
+    using force_new_connection = bool_class<force_new_tag>;
```

Comment:
```suggestion
    using force_new_connection = bool_class<struct force_new_tag>;
```

Reply:
Mm, yes, looks shorter. Is it c++17 compatible?

Reply:
see https://eel.is/c++draft/basic.def#4